src/RouterSelector.native.js

```diff
@@ -11,13 +10,15 @@ import { APP_OPEN, fireEvent, initAnalytics, SIGNIN_FAILED, SIGNIN_SUCCESS } fro
 import Config from './config/config'
 import restart from './lib/utils/restart'
 import DeepLinking from './lib/utils/deepLinking'
+import { savePathToStorage } from './RouterSelectorHelper'
```

Comment:
lets create this as a hook and put it in the hooks folder

Reply:
@sirpy that's not actually a hook, and it's not used inside components
it's some kind of utils and should be added to the `lib/utils`